ext/spl/tests/ArrayObject_overloaded_SplFixedArray.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 SplFixedArray properties is compatible with ArrayObject
+--XFAIL--
+One of the objects is leaking and not being released?
```

Comment:
Using the propert guard somehow makes this case leak and I don't understand why, any ideas @arnaud-lb ?

Reply:
`obj->handlers->write_property()` will ADDREF the value, so you should not `Z_TRY_ADDREF_P(value)` before `obj->handlers->write_property()`